includes/wc-webhook-functions.php

```diff
@@ -128,20 +129,26 @@ function wc_get_webhook_statuses() {
  *
  * @since  3.3.0
  * @throws Exception If webhook cannot be read/found and $data parameter of WC_Webhook class constructor is set.
+ * @param  string   $status Optional - status to filter results by. Must be a key in return value of @see wc_get_webhook_statuses(). @since 3.5.0.
+ * @param  null|int $limit Limit number of webhooks loaded. @since 3.6.0.
  * @return bool
  */
-function wc_load_webhooks() {
+function wc_load_webhooks( $status = '', $limit = null ) {
 	$data_store = WC_Data_Store::load( 'webhook' );
-	$webhooks   = $data_store->get_webhooks_ids();
-	$loaded     = false;
+	$webhooks   = $data_store->get_webhooks_ids( $status );
+	$loaded     = 0;
 
 	foreach ( $webhooks as $webhook_id ) {
 		$webhook = new WC_Webhook( $webhook_id );
 		$webhook->enqueue();
-		$loaded = true;
+		$loaded ++;
+
+		if ( $loaded >= $limit ) {
```

Comment:
If you want the default to be unlimited this would need to be

`if ( null !== $limit && $loaded >= $limit ) {`

Reply:
Ah you're right, thanks!